src/apps/vpn/platforms/android/androidutils.cpp

```diff
@@ -29,22 +29,54 @@ constexpr auto UTILS_CLASS = "org/mozilla/firefox/vpn/qt/VPNUtils";
 }  // namespace
 
 // static
-QString AndroidUtils::GetDeviceName() {
+QString AndroidUtils::getDeviceName() {
+  auto name = readStaticString("android/os/Build", "DEVICE");
+  if (name.isNull()) {
+    return QString("Unknown Device");
```

Comment:
Previously this returned "Android Device" instead of "Unknown Device". It this difference OK?